hash/dc.xml

```diff
@@ -775,15 +782,16 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 		</part>
 	</software>
 
-	<!-- This is labeled "Rev. A". It is unknown if there ever was a "Rev. 0" released. -->
 	<software name="adaisue">
-		<!-- http://redump.org/disc/41732/
+		<!-- This is labeled "Rev. A". It is unknown if there ever was a "Rev. 0" released. -->
+		<!-- http://redump.org/disc/41732/ -->
+		<!--
 		<rom name="Advanced Daisenryaku - Sturm ueber Europa - Der Deutsche Blitzkrieg (Japan).gdi" size="333" crc="356a822e" md5="0d7c41b2625ae226ac0545775ca885be" sha1="3b443572cbd2c505ad6f4e81a539d2daafbafa60"/>
 		<rom name="Advanced Daisenryaku - Sturm ueber Europa - Der Deutsche Blitzkrieg (Japan) (Track 1).bin" size="3843168" crc="bff72044" md5="42982e5e5469cfa18aa20151d8b241d8" sha1="1c01362b7c792bad8fed26ba5e8923b4b343239a"/>
 		<rom name="Advanced Daisenryaku - Sturm ueber Europa - Der Deutsche Blitzkrieg (Japan) (Track 2).bin" size="2088576" crc="1859ea65" md5="652a31166a58b8f933e8a924e7c88780" sha1="0ecd537e42946f3e74c80c9fcb8d0bccccfc96f1"/>
 		<rom name="Advanced Daisenryaku - Sturm ueber Europa - Der Deutsche Blitzkrieg (Japan) (Track 3).bin" size="1185760800" crc="d93bc2f6" md5="bc9b8f7e9d15964387aa45fff09c7092" sha1="f1a8a226138fb642c7ad1bf977a55e19649a50e2"/>
 		-->
-		<description>Advanced Daisenryaku: Sturm über Europa - Der deutsche Blitzkrieg (Japan)</description>
+		<description>Advanced Daisenryaku - Sturm ueber Europa - Der Deutsche Blitzkrieg (Japan)</description>
```

Comment:
https://segaretro.org/Advanced_Daisenryaku:_Sturm_%C3%BCber_Europa_-_Der_deutsche_Blitzkrieg
As per 18 Wheeler plus removing umlaut plus deutsche is lowercase on title/boxart (last part should be a comment).

Reply:
This and the ':' from the previous comment were following redump convention which appears to be ascii and avoids characters which cause filesystem issues.  These of course aren't an issue if this field is just to be a descriptive text field, but then some other field would be needed to match against a source.

Reply:
Not sure I'm following: why you're trying to change the descriptions for making a script to be able to tell what redump.org is gonna tell you by scraping the website instead and by analyzing the filename that is inside the entry? Whatever logic you are following is extremely busted as a data standpoint.

Reply:
The redump URLs in the script are mostly from past maintainers, although I probably added a few as I was researching how the script might work, and I do some scraping in the script I'm working on, but only interactively to show a user useful information to confirm a software list entry is a genuine match to a specific redump title.

The basic idea is:
 - Start with a redump DAT file combined with the actual dumps
 - Parse the software list
 - Run through all the software list descriptions to find matches/close matches to redump DAT titles - auto-confirm where possible, otherwise query the user for the correct match - if a redump URL is available it does scrape to gather some info and romanise Japanese titles, etc to help the user determine a genuine match

Next step is to put all of these mappings somewhere in the Software List itself (ideally).  I was thinking the description field might be ok as it's not particularly well maintained with any standard convention, hence this push - but based on the number of objections maybe that's not the best approach.

Once the mapping is done it becomes trivial to write a script to build new CHDs from the redump dat/dumps from the first step.  Theoretically this could be extended to TOSEC as well, but TOSEC naming convention doesn't fit the description field at all, it would definitely need some sort of dedicated set of 'sourcedump' field(s).

The redump URL while good for research/confirmation isn't really valuable for creating CHD's - in fact redump's DAT file titles don't generally match their website because it's stored in the website's own database a bit differently.